arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Iterable.kt

```diff
@@ -793,11 +793,14 @@ public inline fun <A, B, C> Iterable<C>.unzip(fc: (C) -> Pair<A, B>): Pair<List<
  * ```
  * <!--- KNIT example-iterable-13.kt -->
  */
-@Deprecated(
-  "The current unalign function is renamed to separateIor, and a new unalign function is going to be added to Arrow 2.0.0.",
-  ReplaceWith("separateIor()", "arrow.core.separateIor")
-)
-public fun <A, B> Iterable<Ior<A, B>>.unalign(): Pair<List<A>, List<B>> = separateIor()
+public fun <A, B> Iterable<Ior<A, B>>.unalign(): Pair<List<A?>, List<B?>> =
+  fold(emptyList<A>() to emptyList()) { (l, r), x ->
+    x.fold(
+      { l + it to r + null},
```

Comment:
I prefer using the `Pair` constructor instead of `to` when the pair represents simply two things put together, as opposed to a map entry where one is the key and the other the value.

Reply:
i have changed it @serras

Reply:
Thanks! :)